tools/etcd-dump-db/main.go

```diff
@@ -107,6 +114,27 @@ func iterateBucketCommandFunc(_ *cobra.Command, args []string) {
 	}
 }
 
+func scanKeysCommandFunc(_ *cobra.Command, args []string) {
+	if len(args) != 2 {
+		log.Fatalf("Must provide 2 arguments (got %v)", args)
+	}
```

Comment:
Suggestion to use the `Args` field of `cobra.Command` (in line 44)

```
Args:  cobra.ExactArgs(2),
```

However, I see that other command functions have the validation defined this way.

Reply:
Yes, I intentionally kept it consistent with other commands. Please feel free to update them separately. Thanks.